samples/sensor/max31865/README

```diff
@@ -0,0 +1,3 @@
+This is a sample app to read a MAX31865.
```

Comment:
This file should be in .rst format. See `samples/hello_world/README.rst` for example